scrapers/SpankBang.yml

```diff
@@ -0,0 +1,31 @@
+name: "SpankBang"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - spankbang.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class='left']/h1/@title
+      Details: //meta[@name='description']/@content
+      Image: //div[@class='play_cover']/img/@src
+      Studio:
+        Name:
+          selector: //li[@class='us']/a[@class='ul']/@href[contains(.,'channel')]
+          replace:
+            - regex: ^
+              with: https://spankbang.com
+          subScraper:
+            selector: //h1
+            postProcess:
+              - replace:
+                - regex: (.+)\sOfficial.+
+                  with: $1
+      Tags:
+        Name: //div[@class='ent']/a[@href[contains(.,'category') or contains(.,'tag')]]/text()|//li[@class='p']/text()
```

Comment:
From the samples i tested it seems that tags always contain the keywords from category and we get a lot of dupes as a result. Was there a case where the `category` was needed ?

Reply:
I didn't notice the repetition on that one, looking at the examples I was referencing that seems to stay true. I'll remove the category